package.json

```diff
@@ -8,7 +8,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "test": "mocha test/test.js"
+    "test": "mocha test/test.js && tsc -p test-types"
   },
   "keywords": [
```

Comment:
Please also add a 'types' field pointing to index.d.ts.  Even though this is the default, according to https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html it is still advisable to have the field in.

```
Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so.
```